### PR TITLE
 [DO-2132] [Contains Migration] Added model for family members 

### PR DESCRIPTION
**Pull request by EmmaSamoilova**
NOT FOR MERGE.

### CODE REVIEW

File: business_register/models/declaration_models.py [hunk 1]
```diff
@@ -1403,6 +1451,15 @@ class BaseRight(DataOceanModel):
         default=None,
         help_text='politically exposed person that owns the right'
     )
+    owner_fm = models.ForeignKey(
```
[the remark this answers is not in the document]
Comment (EmmaSamoilova) [reply business_register/models/declaration_models.py]: done

File: business_register/models/declaration_models.py [hunk 2]
```diff
@@ -1321,6 +1311,64 @@ class Property(DataOceanModel):
...
+        (ROOMMATE, 'persons who live together but are not married'),
+    )
+    declaration = models.ForeignKey(
+        Declaration,
+        on_delete=models.CASCADE,
+        related_name='family_members',
+        verbose_name='declaration',
+        help_text='a declaration in which a family member is indicated'
+    )
+    pep_id = models.ForeignKey(
```
Comment (roman-tiukh) [inline business_register/models/declaration_models.py:1338]: pep

File: business_register/models/declaration_models.py [hunk 3]
```diff
@@ -1321,6 +1311,64 @@ class Property(DataOceanModel):
...
+        Pep,
+        on_delete=models.CASCADE,
+        verbose_name='pep id',
+        help_text='from declarant link'
+    )
+    nacp_id = models.PositiveBigIntegerField(
```
Comment (roman-tiukh) [inline business_register/models/declaration_models.py:1344]: ArrayField?

File: business_register/models/declaration_models.py [hunk 4]
```diff
@@ -1321,6 +1311,64 @@ class Property(DataOceanModel):
...
 
+class FamilyMember(DataOceanModel):
+    WIFE = 1
+    HUSBAND = 2
+    DAUGHTER = 3
+    SON = 4
+    FATHER = 5
+    MOTHER = 6
+    ROOMMATE = 7
+    RELATIONSHIP_TYPE = (
```
Comment (roman-tiukh) [inline business_register/models/declaration_models.py:1322]:
```suggestion
    RELATIONSHIP_TYPES = (
```

File: business_register/models/declaration_models.py [hunk 5]
```diff
@@ -1321,6 +1311,64 @@ class Property(DataOceanModel):
...
+        db_index=True,
+    )
+    relationship_type = models.SmallIntegerField(
+        'relationship_type',
+        choices=RELATIONSHIP_TYPE,
+        help_text='the type of relationship specified in the declaration',
+    )
+    first_name = models.CharField(
+        'first name',
+        max_length=20,
```
Comment (roman-tiukh) [inline business_register/models/declaration_models.py:1355]:
```suggestion
        max_length=30,
```

File: business_register/models/declaration_models.py [hunk 6]
```diff
@@ -1321,6 +1311,64 @@ class Property(DataOceanModel):
...
+        help_text='First name of declarant`s family member',
+    )
+    middle_name = models.CharField(
+        'middle name',
+        max_length=25,
```
Comment (roman-tiukh) [inline business_register/models/declaration_models.py:1360]:
```suggestion
        max_length=100,
```

File: business_register/models/declaration_models.py [hunk 7]
```diff
@@ -1321,6 +1311,64 @@ class Property(DataOceanModel):
...
+        'NACP id',
+        db_index=True,
+    )
+    relationship_type = models.SmallIntegerField(
+        'relationship_type',
+        choices=RELATIONSHIP_TYPE,
+        help_text='the type of relationship specified in the declaration',
+    )
+    first_name = models.CharField(
+        'first name',
```
Comment (roman-tiukh) [inline business_register/models/declaration_models.py:1354]: verbose name not needed
```suggestion
```

File: business_register/models/declaration_models.py [hunk 8]
```diff
@@ -1321,6 +1311,64 @@ class Property(DataOceanModel):
...
+        max_length=20,
+        help_text='First name of declarant`s family member',
+    )
+    middle_name = models.CharField(
+        'middle name',
```
Comment (roman-tiukh) [inline business_register/models/declaration_models.py:1359]: verbose name not needed
```suggestion
```

File: business_register/models/declaration_models.py [hunk 9]
```diff
@@ -1321,6 +1311,64 @@ class Property(DataOceanModel):
...
+        max_length=25,
+        default='',
+        blank=True,
+        help_text='Middle name of declarant`s family member',
+    )
+    last_name = models.CharField(
+        'last name',
```
Comment (roman-tiukh) [inline business_register/models/declaration_models.py:1366]: verbose name not needed
```suggestion
```

File: business_register/models/declaration_models.py [hunk 10]
```diff
@@ -1321,6 +1311,64 @@ class Property(DataOceanModel):
...
+    RELATIONSHIP_TYPE = (
+        (WIFE, 'wife'),
+        (HUSBAND, 'husband'),
+        (DAUGHTER, 'daughter'),
+        (SON, 'son'),
+        (FATHER, 'father'),
+        (MOTHER, 'mother'),
+        (ROOMMATE, 'persons who live together but are not married'),
+    )
+    declaration = models.ForeignKey(
```
Comment (roman-tiukh) [inline business_register/models/declaration_models.py:1331]: @IegorSoboliev do we need to tie them to the declaration?

**Review by roman-tiukh — COMMENTED** [review]
how about the relation from liability?

**Comment by EmmaSamoilova** [comment]
> @EmmaSamoilova what's status of this PR?

probably no longer actual. I close

**Pull request closed without merging**